src/App.js

```diff
@@ -1,16 +1,110 @@
 import React from 'react';
+import TodoList from './components/TodoComponents/TodoList';
+import TodoForm from './components/TodoComponents/TodoForm';
+
+  const listData = [
+    {
+      task: 'Organize Garage',
+      id: 1528817077286,
+      completed: false
+    },
+    {
+      task: 'Bake Cookies',
+      id: 1528817084358,
+      completed: false
+    }
+  ];
 
 class App extends React.Component {
   // you will need a place to store your state in this component.
   // design `App` to be the parent component of your application.
   // this component is going to take care of state, and any change handlers you need to work with your state
+  constructor(){
+    super();
+    this.state = {
+      itemList: listData,
+      inputText: '',
+    }
+  }
+
+  handleChange = ev =>{
+    this.setState({
+      [ev.target.name]: ev.target.value
```

Comment:
Consider [object destructuring](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment#Object_destructuring) here to make this easier to write and read:
```javascript
const { name, value } = ev.target;
this.setState({ [name]: value });
```